repositories/supplier.go

```diff
@@ -1,3 +1,4 @@
+//go:generate mockgen -source=supplier.go -destination=../repositories/mocks/mock_supplier_repository.go -package=mock
```

Comment:
I guess we should use only singular names of folders. Like in Golang code-style.
'mock' instead of -> 'mocks'